doc/source/guide/primal_dual_hybrid_gradient_guide.rst

```diff
@@ -1,10 +1,10 @@
-.. _chambolle_pock_guide:
+.. _primal_dual_hybrid_gradient_guide:
 
 #####################
-Chambolle-Pock solver
+Primal-Dual Hybrid Gradient (PDHG) solver
 #####################
 
-The `chambolle_pock_solver` was introduced in 2011 by Chambolle and Pock in the paper `A first-order primal-dual algorithm for convex problems with applications to imaging
+The `primal_dual_hybrid_gradient_solver` was studied in 2011 by Chambolle and Pock in the paper `A first-order primal-dual algorithm for convex problems with applications to imaging
```

Comment:
I'd scrap `_solver` from the function name now. Too long as is.

Perhaps we could even go with `pdhg`?